lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -408,6 +413,17 @@ protected function getLanguage() {
   }
 
   /**
+   * Get campaign's latest news.
+   *
+   * @return array
+   */
+  protected function getLatestNews() {
+    return [
+      'latest_news' => dosomething_helpers_extract_field_data($this->node->field_latest_news_copy),
```

Comment:
Is there a way to test what the would return if it has sample data in it? Cause if it returns an array of stuff, those may need to be further processed... I could envision the `latest_news` property being an array of latest news objects with their own data...